_slides/14-toddwalstad-eaton.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Most people use and IDE that hides all this, this is good to learn what is happening under the hood!
```

Comment:
That's good

Reply:
@ViktorStamenov-eaton thanks, can you please approve?